server/models/migrations/next/migrate.sql

```diff
@@ -1 +1,56 @@
+<<<<<<< HEAD
```

Comment:
This is a broken merge that needs to be fixed.

Otherwise, this looks good to me.

Reply:
Hooo